www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -793,6 +793,29 @@
         <default>0</default>
         <Required>Y</Required>
       </https_only>
+      <tls_protocols type="OptionField">
+        <multiple>Y</multiple>
+        <Sorted>Y</Sorted>
+        <OptionValues>
+          <TLSv1_2 value="TLSv1.2">TLSv1.2</TLSv1_2>
+          <TLSv1_3 value="TLSv1.3">TLSv1.3</TLSv1_3>
+        </OptionValues>
+        <Required>N</Required>
+        <default>TLSv1.2,TLSv1.3</default>
+      </tls_protocols>
+      <tls_ciphers type="TextField">
+        <default>ECDHE-ECDSA-CAMELLIA256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-CAMELLIA256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-CAMELLIA128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-CAMELLIA128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-CAMELLIA256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-CAMELLIA256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-ECDSA-CAMELLIA128-SHA256:ECDHE-RSA-AES128-SHA256</default>
+        <Required>N</Required>
+        <mask>/^((((!|\+|-)?[A-Z][A-Z\d\+-]+)|(@STRENGTH)):?)*$/i</mask>
+      </tls_ciphers>
+      <tls_ecdh_curve type="TextField">
+        <Required>N</Required>
+        <mask>/^(([A-Z\d-]+):?)*$/i</mask>
+      </tls_ecdh_curve>
+      <tls_prefer_server_ciphers type="BooleanField">
+        <default>1</default>
+        <Required>Y</Required>
```

Comment:
oh, also needs a version bump in the model

Reply:
sorry, I thought that version bumped in 
https://github.com/opnsense/plugins/blob/ee12197aca53bf9b9f8cd6811ec29214c2dac362/www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml#L3
and removed it here. should i bump it anyway?

Reply:
fair point, no need to do it again then